sdks/python/apache_beam/utils/annotations.py

```diff
@@ -90,6 +90,8 @@ def inner(*args, **kwargs):
       message = '%s is %s' % (fnc.__name__, label)
```

Comment:
Yes, and It is possible, as long as `since` is present for deprecated (the rest is optional).